panasonic_camera/camera_manager.py

```diff
@@ -47,6 +47,9 @@ def _connect(self):
             logger.debug(
                 'Connect to {}: {}'.format(device.friendly_name, hostname))
             self.camera = PanasonicCamera(hostname)
+            # Some cameras like Panasonic DC-FZ80 require registering your
+            # device with the camera first
+            self.camera.register_with_camera()
```

Comment:
Can you call

```python
self.camera.get_info_capability()
```

(see statement below) before you register the camera? What does your DC-FZ80 return as capability? Maybe it indicates if the `accctrl` request is required.

Reply:
Whenever I do `self.camera.get_info_capability()`, the DC-FZ80 always responds:
`<?xml version="1.0" encoding="UTF-8"?>
<camrply><result>err_unsuitable_app</result></camrply>`. If I do this before `accctrl`, it causes a message on the camera screen says that the smartphone app needs to be updated. However, it doesn't prevent me from doing an `accctrl` subsequently and establishing remote control. `_validate_camrply` throws an exception with the `err_unsuitable_app` response, but I supressed that and confirmed that the camera still accepts remote control even if the `accctrl` request comes right after a get_info_capability() request.

Reply:
Hm. So I think there's some additional complexity here -- the request / response flow may be different on first connection. I tried "Reset Wi-Fi Settings" on my camera and the first time I connect to it after that, I'm forced to confirm the connection on the camera screen with a Yes / No dialog. For determining what counts as the first time, it's not using the name sent during accctrl... must be using IP address or something.

Reply:
Will be fixed/improved later (see #42)